core/src/main/scala/kafka/admin/TopicCommand.scala

```diff
@@ -200,13 +200,14 @@ object TopicCommand extends Logging {
           val describeConfigs: Boolean = !reportUnavailablePartitions && !reportUnderReplicatedPartitions
           val describePartitions: Boolean = !reportOverriddenConfigs
           val sortedPartitions = topicPartitionAssignment.toList.sortWith((m1, m2) => m1._1 < m2._1)
+          val markedForDeletion = if (zkUtils.isTopicMarkedForDeletion(topic)) "MarkedForDeletion:true" else ""
           if (describeConfigs) {
             val configs = AdminUtils.fetchEntityConfig(zkUtils, ConfigType.Topic, topic).asScala
             if (!reportOverriddenConfigs || configs.nonEmpty) {
               val numPartitions = topicPartitionAssignment.size
               val replicationFactor = topicPartitionAssignment.head._2.size
-              println("Topic:%s\tPartitionCount:%d\tReplicationFactor:%d\tConfigs:%s"
-                .format(topic, numPartitions, replicationFactor, configs.map(kv => kv._1 + "=" + kv._2).mkString(",")))
+              println("Topic:%s\tPartitionCount:%d\tReplicationFactor:%d\tConfigs:%s\t%s"
```

Comment:
I wonder if it would be better to move `\t` inside `markedForDeletion` so `\t` won't be printed if topic is not marked for deletion (similar to how it is handled in line 225 below). Also, the PR needs a rebase.